augmentedVision/rectangle[] rectanglesLeft
augmentedVision/rectangle[] rectanglesRight